src/Bswup/Demos/FullDemo/Client/Shared/DemoBswupProgressBar.razor.css

```
@@ -30,21 +30,21 @@
-    top: 2px;
-    left: 50%;
     display: none;
-    z-index: 999999;
     position: fixed;
-    text-align: center;
+    left: 50%;
+    top: 2px;
     transform: translateX(-50%);
+    text-align: center;
+    z-index: 999999;
     font-family: "Segoe UI", "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;
```

Stylelint (17.14.0)

[error] Duplicate font-family name "Segoe UI" (font-family-no-duplicate-names)

(font-family-no-duplicate-names)